sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultReadinessCheckSpec.java

```diff
@@ -0,0 +1,135 @@
+        Builder builder = new Builder();
+        builder.command = copy.command;
+        builder.delay = copy.delay;
+        builder.interval = copy.interval;
+        builder.timeout = copy.timeout;
+        return builder;
+    }
+
+    @Override
+    public String getCommand() {
+        return command;
+    }
+
+    @Override
+    public Integer getDelay() {
+        return delay;
+    }
+
+    @Override
+    public Integer getInterval() {
+        return interval;
+    }
+
+    @Override
+    public Integer getTimeout() {
+        return timeout;
+    }
+
+    /**
+     * {@code DefaultReadinessCheckSpec} builder static inner class.
+     */
+    public static final class Builder {
+        private String command;
+        private Integer delay;
+        private Integer interval;
+        private Integer timeout;
+
+        private Builder() {
+        }
+
+        /**
+         * Sets the {@code command} and returns a reference to this Builder so that the methods can be chained together.
+         *
+         * @param command the {@code command} to set
+         * @return a reference to this Builder
+         */
+        public Builder command(String command) {
+            this.command = command;
+            return this;
+        }
+
+        /**
+         * Sets the {@code delay} and returns a reference to this Builder so that the methods can be chained together.
+         *
+         * @param delay the {@code delay} to set
+         * @return a reference to this Builder
+         */
+        public Builder delay(Integer delay) {
+            this.delay = delay;
+            return this;
+        }
+
+        /**
+         * Sets the {@code interval} and returns a reference to this Builder so that the methods can be chained
+         * together.
+         *
+         * @param interval the {@code interval} to set
+         * @return a reference to this Builder
+         */
+        public Builder interval(Integer interval) {
+            this.interval = interval;
+            return this;
+        }
+
+        /**
+         * Sets the {@code timeout} and returns a reference to this Builder so that the methods can be chained together.
+         *
+         * @param timeout the {@code timeout} to set
+         * @return a reference to this Builder
+         */
+        public Builder timeout(Integer timeout) {
+            this.timeout = timeout;
+            return this;
+        }
+
+        /**
+         * Returns a {@code DefaultHealthCheckSpec} built from the parameters previously set.
+         *
+         * @return a {@code DefaultHealthCheckSpec} built with parameters of this {@code DefaultHealthCheckSpec.Builder}
+         */
+        public DefaultReadinessCheckSpec build() {
+            DefaultReadinessCheckSpec defaultReadinessCheckSpec = new DefaultReadinessCheckSpec(this);
+            ValidationUtils.validate(defaultReadinessCheckSpec);
+            return defaultReadinessCheckSpec;
+        }
+    }
+}
```

Comment:
Define `equals`, `hashCode`, `toString`? I think those will be necessary for verifying serialization/deserialization should we ever need to test those.